<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nhap_dausach</class>
 <widget class="QDialog" name="nhap_dausach">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>180</y>
     <width>195</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QPushButton" name="pushButton_nhap">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>-90</x>
     <y>20</y>
     <width>731</width>
     <height>136</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>108</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_ISBN">
      <property name="styleSheet">
       <string notr="true">font-size: 15px;color:black;font-weight:700;
</string>
      </property>
      <property name="text">
       <string>ISBN </string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLineEdit" name="lineEdit_ISBN_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maxLength">
       <number>17</number>
      </property>
      <property name="placeholderText">
       <string>ISBN-10 / ISBN-13</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label__theloai_6">
      <property name="styleSheet">
       <string notr="true">font-size: 15px;color:black;font-weight:700;</string>
      </property>
      <property name="text">
       <string>Thể loại</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6" colspan="4">
     <widget class="QLineEdit" name="lineEdit_theloai_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gray;</string>
      </property>
      <property name="placeholderText">
       <string>Science,realistic,...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QLabel" name="label_namxb_5">
      <property name="styleSheet">
       <string notr="true">font-size: 15px;color:black;font-weight:700;</string>
      </property>
      <property name="text">
       <string>năm xuất bản</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QSpinBox" name="spinBox_namxb_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gray;</string>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="label_tacgia_4">
      <property name="styleSheet">
       <string notr="true">font-size: 15px;color:black;font-weight:700;</string>
      </property>
      <property name="text">
       <string>Tác giả</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7" colspan="3">
     <widget class="QLineEdit" name="lineEdit_tacgia_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gray;</string>
      </property>
      <property name="placeholderText">
       <string>author</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>98</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_vitri_7">
      <property name="styleSheet">
       <string notr="true">font-size: 15px;color:black;font-weight:700;</string>
      </property>
      <property name="text">
       <string>Vị trí </string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QComboBox" name="comboBox_vitri_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="minimumContentsLength">
       <number>10</number>
      </property>
      <property name="placeholderText">
       <string>A-Z</string>
      </property>
      <item>
       <property name="text">
        <string>A</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>B</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>C</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>D</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>E</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>F</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>G</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>H</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>I</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>J</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>K</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>L</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>M</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>N</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>O</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>P</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Q</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>R</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>S</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>T</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>U</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>V</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>W</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>X</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Y</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Z</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="5" colspan="2">
     <widget class="QLabel" name="label_soluong_8">
      <property name="styleSheet">
       <string notr="true">font-size: 15px;color:black;font-weight:700;</string>
      </property>
      <property name="text">
       <string>Số lượng</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QSpinBox" name="spinBox_soluong_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
     </widget>
    </item>
    <item row="2" column="8">
     <widget class="QLabel" name="label_sotrang_3">
      <property name="styleSheet">
       <string notr="true">font-size: 15px;color:black;font-weight:700;</string>
      </property>
      <property name="text">
       <string>Số trang</string>
      </property>
     </widget>
    </item>
    <item row="2" column="9">
     <widget class="QSpinBox" name="spinBox_sotrang_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gray;</string>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>88</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QLabel" name="label_tensach_2">
      <property name="styleSheet">
       <string notr="true">font-size: 15px;color:black;font-weight:700;</string>
      </property>
      <property name="text">
       <string>Tên sách</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4" colspan="6">
     <widget class="QLineEdit" name="lineEdit_tensach_2">
      <property name="styleSheet">
       <string notr="true">color: gray;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
